subprojects/core-api/src/main/java/org/gradle/api/provider/Provider.java

```diff
@@ -54,7 +54,7 @@
  *     <li>Calling {@link #map(Transformer)} to create a new provider from an existing provider.</li>
  *     <li>Using the return value of {@link org.gradle.api.tasks.TaskContainer#register(String)}, which is a provider that represents the task instance.</li>
  *     <li>Using the methods on {@link org.gradle.api.file.Directory} and {@link org.gradle.api.file.DirectoryProperty} to produce file providers.</li>
- *     <li>By calling {@link ProviderFactory#provider(Callable)} or {@link org.gradle.api.Project#provider(Callable)} to create a new provider from a {@link Callable}.</li>
+ *     <li>By calling {@link ProviderFactory#provider(Callable)}, {@link org.gradle.api.initialization.Settings#provider(Callable)}, or {@link org.gradle.api.Project#provider(Callable)} to create a new provider from a {@link Callable}.</li>
```

Comment:
Once `Settings.provider {}` removed this should be updated. Maybe as a separate bullet point.

Reply:
Removed `Settings.provider(Callable)` as well as this reference.  I didn't add anything else to the Javadocs in `Provider` as I couldn't think of anything relevant to `Settings.getProviders()` - it's more relevant in `ProviderFactory`.  Thoughts?

Reply:
LGTM